docs/pages/development/installation.md

```diff
@@ -53,8 +53,8 @@ Run the following command to install native code for the Dev Launcher via CocoaP
 
 <InstallSection packageName="expo-development-client" cmd={["npx pod-install"]} hideBareInstructions />
 
-Also, make sure that your project is configured to deploy on an iOS version _above 10_.
-To do that, open Xcode and go to General > Deployment Info, and select an iOS version of at least 11.0.
+Also, make sure that your project is configured to deploy on an iOS version of _at least 12_.
+To do that, open Xcode and go to General > Deployment Info, and select an iOS version of at least 12.0.
```

Comment:
```suggestion
Make sure that your project is configured to deploy on an iOS version of _at least 12_. To do that, open Xcode, go to General > Deployment Info, and select an iOS version of at least 12.0.
```

I think this is an existing formatting issue on this page. (for the second line it was using new line in the markdown but rendering in the same paragraph). Not related to the changes you made 👍